ci/github-script/merge.js

```diff
@@ -22,14 +24,14 @@ function runChecklist({
         .reduce((acc, cur) => acc?.intersection(cur) ?? cur)
 
   const checklist = {
-    'PR targets one of the allowed branches: master, staging, staging-next.': [
-      'master',
-      'staging',
-      'staging-next',
-    ].includes(pull_request.base.ref),
+    'PR targets a [development branch](https://github.com/NixOS/nixpkgs/blob/-/ci/README.md#branch-classification).':
```

Comment:
Nit: if we want an absolute URL, it'd be slightly better to avoid hard-coding it and instead construct it from the `github` context.


```suggestion
    `PR targets a [development branch](${context.github.server_url}/${context.github.repository}/blob/-/ci/README.md#branch-classification).`:
```

Reply:
We have another link further down to the nixpkgs-maintainers team. This is also hardcoded to the upstream NixOS org. I think this makes sense, because when you're running this in a fork, you don't necessarily have the same teams set up etc.

Having the links point to the correct thing, even when testing in a fork, is a good thing, imho.